lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoDoc.kt

```diff
@@ -0,0 +1,178 @@
+                            .substringAfter("*")
+                            .removePrefix(" ")
+                    }
+                    .toMutableList()
+                    .also { line -> line.removeIf { element -> element.isBlank() } }
+            }
+
+        firsts + others
+    }
+
+    val paramBlockTags by lazy {
+        val tag = this.tags.filter { it.startsWith("@param") }
+
+        tag.map { parseToValuedBlockTag(it) }
+    }
+
+    val returnBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@return") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val constructorBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@constructor") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val receiverBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@receiver") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val propertyBlockTags by lazy {
+        val tag = this.tags.filter { it.startsWith("@property") }
+
+        tag.map { parseToValuedBlockTag(it) }
+    }
+
+    val throwsBlockTags by lazy {
+        val tag = this.tags.filter { it.startsWith("@throws") || it.startsWith("@exception") }
+
+        tag.map { parseToValuedBlockTag(it) }
+    }
+
+    val sampleBlockTags by lazy {
+        val tag = this.tags.filter { it.startsWith("@sample") }
+
+        tag.map { parseToValuedBlockTag(it) }
+    }
+
+    val seeBlockTags by lazy {
+        val tag = this.tags.filter { it.startsWith("@see") }
+
+        tag.map { parseToValuedBlockTag(it) }
+    }
+
+    val authorBlockTags by lazy {
+        val tag = this.tags.filter { it.startsWith("@author") }
+
+        tag.map { parseToBlockTag(it) }
+    }
+
+    val sinceBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@since") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val suppressBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@suppress") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val versionBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@version") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val propertySetterBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@propertySetter") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val propertyGetterBlockTag by lazy {
+        val tag = this.tags.firstOrNull { it.startsWith("@propertyGetter") }
+
+        tag?.let { parseToBlockTag(it) }
+    }
+
+    val blockTags by lazy {
+        (
+            paramBlockTags + returnBlockTag + constructorBlockTag + receiverBlockTag +
+                propertyBlockTags + throwsBlockTags + sampleBlockTags +
+                seeBlockTags + authorBlockTags + sinceBlockTag + suppressBlockTag +
+                versionBlockTag + propertySetterBlockTag + propertyGetterBlockTag
+            ).filterNotNull()
+    }
```

Comment:
`blockTags` - czyli zbiór wszystkich tag, ale minus tego jest taki, że jak dodamy nowy tag, to musimy tutaj wpisać ręcznie, co jest uciążliwe i łatwo zapomnieć